helm_deploy/values-dev.yaml

```diff
@@ -1,10 +1,9 @@
 ---
 # Environment specific values, override helm_deploy/court-register/values.yaml
 generic-service:
-  replicaCount: 1
+  replicaCount: 2
   ingress:
     host: offender-events-ui-dev.prison.service.justice.gov.uk
-    contextColour: green
```

Comment:
setting was removed a long while ago from the generic service helm chart so removing here too